### PR TITLE
Livechat new layout

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

I've moved the "custom fields" form to to it's own template. So now I'm calling it from the register page _and_ on "My Account" page, so users can fill/update the custom fields after registration.

Livechat widget is now bigger (both height and width) to display the agent informantion fixed on the header. It's also showing the agent's **name** on messages.

Now:
image

Before:
image

### CODE REVIEW

File: packages/rocketchat-theme/assets/stylesheets/base.less [hunk 1]
```diff
@@ -4031,10 +4031,6 @@ body:not(.is-cordova) {
 			}
 		}
 
-		.select-arrow {
```
Comment (rodrigok) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: Why did you remove this arrow?

Comment (sampaiodiego) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: because we don't have an arrow on any other `<select/>`

Comment (rodrigok) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: Yes, we have, on admin page

Comment (sampaiodiego) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: true. the only one left :stuck_out_tongue:

Comment (karlprieb) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: As we discussed before I will add `select-arrow` to all `select` and make it global in another PR.

Comment (rodrigok) [inline packages/rocketchat-theme/assets/stylesheets/base.less]: @karlprieb thanks for keeping me updated
